README.md

```diff
@@ -1,56 +1,62 @@
 [![REUSE status](https://api.reuse.software/badge/github.com/SAP/ai-sdk-js)](https://api.reuse.software/info/github.com/SAP/ai-sdk-js)
 [![Fosstars security rating](https://github.com/SAP/ai-sdk-js/blob/fosstars/fosstars_badge.svg)](https://github.com/SAP/ai-sdk-js/blob/fosstars/fosstars_report.md)
 
-# ai-sdk-js
+# SAP AI SDK
```

Comment:
Question to @jjtang1985, do we call it "SAP Cloud SDK for AI" here?

Reply:
I guess I can extend this to some other sections as well like our "About" Section on the entry page, I guess making the naming clear would be helpful
( SAP Cloud SDK for AI, for Gen AI Hub, for Ai Core)

Reply:
Yes, please use the official name in any prosa text and titles.